.github/workflows/stale.yaml

```diff
@@ -20,6 +20,7 @@ jobs:
       - uses: actions/stale@v5
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+          operations-per-run: 60
```

Comment:
Can we keep using the default value (30) here? My concern is that by increasing this value to 60, other jobs failed to run due to the GitHub API limit.

WDYT?

Reply:
Sure, sounds good. Let's keep the default value for now since we are going to run this action every 5th hour.

Reply:
If any problem occurs, re-thinking the parameters would be great :)